### DISCUSSION TITLE
demo about sealing on gramine

### DISCUSSION THREAD

**Discussion by RicLee0124** [gramineproject/gramine#1044] [General]
I am learnig about sealing usage on gramine, but I can not found and demo about, I do not konw how to use the manifest syntax 
 . any one cloud supply a demo about sealing on gramine?  great appreciate it !!!

**Comment by dimakuv** [comment]
https://github.com/gramineproject/gramine/tree/master/libos/test/regression -- please check the `sealed_file.c` and corresponding manifest `manifest.template` and how this test is run in `test_libos.py`.

Please also read through https://gramine.readthedocs.io/en/latest/manifest-syntax.html#encrypted-files
